recipes/default.rb

```diff
@@ -207,6 +210,33 @@
   not_if { ::File.exist? librenms_rrddir }
 end
 
+directory librenms_bootstrap_cachedir do
+  owner librenms_username
+  group librenms_group
+  mode '0755'
+  action :create
+  not_if { ::File.exist? librenms_bootstrap_cachedir }
```

Comment:
This is exactly what the resource `directory` does

Reply:
Hopefully fixed